crates/bevy_ecs/src/system/query.rs

```diff
@@ -320,11 +320,11 @@ use std::{any::TypeId, borrow::Borrow};
 /// [`single_mut`]: Self::single_mut
 /// [`SparseSet`]: crate::storage::SparseSet
 /// [System parameter]: crate::system::SystemParam
-/// [`Table`]: crate::storage::Table
 /// [`With`]: crate::query::With
+/// [`Table`]: crate::storage::Table
 /// [`Without`]: crate::query::Without
+// SAFETY: Must have access to the components registered in `state`.
```

Comment:
Can you revert this? The safety comment belongs on the `UnsafeWorldCell` field.

Reply:
Yeah. Not sure how I changed this to be honest.

Reply:
reverted